src/main/java/org/generationcp/middleware/domain/search_request/ObservationUnitsSearchRequestDto.java

```diff
@@ -0,0 +1,101 @@
+package org.generationcp.middleware.domain.search_request;
```

Comment:
we should have two packages? for v1 and v2?

Reply:
@vmaletta please reorganize the search requests in v1 and v2

Reply:
ok